docs/src/tutorial-real-code.md

```diff
@@ -36,7 +36,7 @@ Dependencies can sometimes blow up the tractability of proofs.
 This can usually be handled, but requires a lot more investment to make it happen, and so isn't a good place to start.
 
 2. Don't forget to consider starting with your dependencies.
-Sometimes the best place to start won't be your code, but in code you depend on.
+Sometimes the best place to start won't be your code, but the code you depend on.
```

Comment:
Imo, "on the code that you depend upon" would be a better to phrase this.

Reply:
Updated as "the code that you depend on", because in your suggestion:
 * I think adding "on" changes meaning (the sentence would read "the best place to start will be on the code..." if we remove the "won't be your code, but") because it refers to a more specific location. This is not what we want to express.
 * "on"/"upon" are interchangeable, so I would rather keep "on".